src/AspNetCore.Identity.MongoDB/MongoUserStore.cs

```diff
@@ -71,7 +71,7 @@ public MongoUserStore(IMongoDatabase database, ILoggerFactory loggerFactory, str
 
             cancellationToken.ThrowIfCancellationRequested();
 
-            await _usersCollection.InsertOneAsync(user, cancellationToken).ConfigureAwait(false);
+            await _usersCollection.InsertOneAsync(user, null, cancellationToken).ConfigureAwait(false);
```

Comment:
the previews method was obsoleted with this message => Use the new overload of InsertOneAsync with an InsertOneOptions parameter instead.